Scripts/lint.sh

```
@@ -118,35 +113,47 @@ generate_swiftlint_compiler_log() {
+    analyzed_files=$(find "$package_root/Sources" "$package_root/Tools" -name '*.swift')
+    expected_count=$(printf '%s\n' "$analyzed_files" | wc -l | tr -d ' ')
+    swift_package_plugin swiftlint analyze \
         --strict \
         --force-exclude \
-        --compiler-log-path "$package_root/.build/swiftlint-compiler.log" > "$analyze_log" 2>&1; then
-        # Everything but the per-file progress: skipped files and issues.
-        grep -v -E '^(Collecting|Linting) ' "$analyze_log"
-        echo "SwiftLint analyze passed; full log: $analyze_log"
-        return 0
+        --compiler-log-path "$package_root/.build/swiftlint-compiler.log" \
+        $analyzed_files > "$analyze_log" 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)